### DISCUSSION TITLE
Some questions about blueretro SNES

### DISCUSSION THREAD

**Discussion by Cyrus331** [darthcloud/BlueRetro#231] [Q&A]
Hello, bro. Why can 1p be connected according to this figure, but the key doesn't respond? 2p can be used normally when connected with PS4, corresponding to L2 \ R2. The LR key doesn't respond after connecting with the handle of 8bitdo. How to set the web config of sens system has not been understood. How to set the mapping congfig, SRC label Does it represent my Bluetooth device?

![Usando 2 (74AHC125) LA OPCIÓN MAS RECOMENDADA ](https://user-images.githubusercontent.com/98383215/159119194-3af5edc3-731c-4ae2-bd14-78ee7ffbe294.png)

**Comment by darthcloud** [comment]
Moved to discussion.

Connect your 8bitdo in "X input" mode and it should work by default. Yes source is BT buttons. Use Xbox as source label for 8bitdo in X input mode.

**Reply by Cyrus331** [reply]
There's another question I'd like to ask. Can't I insert the entity handle after I turn on blueretro?, I've tried Wii and SNES, but they didn't respond when they inserted the handle when blueretro was on

**Reply by darthcloud** [reply]
No sure you means by handle.

**Reply by Cyrus331** [reply]
Can't I insert the entity controller after I turn on blueretro?, I've tried Wii and SNES, but they didn't respond when they inserted the controlle when blueretro was on

When 8bitdo is connected first as 1p, no key will respond. At the same time, if 2p is connected to PS4, it will be normal.

**Reply by darthcloud** [reply]
You can't use original controller and Blueretro simultaneously. Not supported.
